### PR TITLE
Assignment1 - Josh Pedersen

### PR DESCRIPTION
**Pull request by Joshua-Pedersen**
Fixed failing test and crashing issue.

### CODE REVIEW

**Comment by rkdeboer** [comment]
Looks wonderful! Great job using a float cast!

**Comment by Seyoumyonas** [comment]
I will review this ,
This is Yonas Seyoum.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -79,13 +79,16 @@ public static Question[] LoadQuestions(string filePath)
...
+            Question question = new()
+            {
+                Text = questionText,
+                Answers = new string[3]
+            };
             question.Answers[0] = answer1;
             question.Answers[1] = answer2;
             question.Answers[2] = answer3;
             question.CorrectAnswerIndex = correctAnswerIndex;
+            questions[i] = question;
```
Comment (Seyoumyonas) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:91]: this is the correct code, the code works as expected. good job. 
Yonas.

**Review by Seyoumyonas — APPROVED** [review]
the code works as expected. Good job. 
Yonas.

**Pull request closed without merging**